src/dev/build/tasks/os_packages/docker_generator/resources/bin/kibana-docker

```diff
@@ -138,7 +138,16 @@ kibana_vars=(
     tilemap.url
     timelion.enabled
     vega.enableExternalUrls
+    xpack.actions.allowedHosts
+    xpack.actions.enabled
+    xpack.actions.enabledActionTypes
+    xpack.actions.preconfigured
     xpack.actions.proxyUrl
+    xpack.actions.proxyHeaders
+    xpack.actions.proxyRejectUnauthorizedCertificates
+    xpack.actions.rejectUnauthorized
+    xpack.alerts.invalidateApiKeysTask.interval
```

Comment:
Not sure if you want to add it - but also missing `xpack.alerts.healthCheck.interval`

Reply:
Thanks!